README.md

```diff
@@ -142,6 +180,52 @@ For example:
 
 The final result of the above command was 10 votes for witness1 and 0 vote for witness2.
 
+## Brokerage
+
+After voting, you will be rewarded with brokerages. Witnesses have the right to decide the ratio of brokerages. The default ratio is 20%, and witnesses can adjust it.
+
+By default, if a witness is rewarded, he will receive 20% of the rewards, and 80% of the rewards will be distributed to his voters.
+
+### getbrokerage
+
+View the ratio of brokerages
+
+    > getbrokerage OwnerAddress
+
+OwnerAddress
+> The address of the witness's account
+
+### getreward
+
+Query unclaimed brokerages
+
+    > getreward OwnerAddress
+    
+OwnerAddress
+> The address of the voter's account
+
+### updateBrokerage
+
+Update the ratio of brokerages
```

Comment:
Update the ratio of brokerage, this command is usually used by a witness account.